### ISSUE TITLE
accessibility signals for voice should be in separate contribution file

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#209487]
https://github.com/microsoft/vscode/blob/32e015a091fcbd0692865814c5e6bed175a7987f/src/vs/workbench/contrib/speech/browser/speechService.ts#L159

cc @bpasero 

similar to https://github.com/microsoft/vscode/blob/32e015a091fcbd0692865814c5e6bed175a7987f/src/vs/workbench/contrib/accessibility/browser/saveAccessibilitySignal.ts#L5-L24
